sentry-android-core/src/main/java/io/sentry/android/core/NdkIntegration.java

```diff
@@ -0,0 +1,35 @@
+package io.sentry.android.core;
+
+import android.os.Build;
+import io.sentry.core.IHub;
+import io.sentry.core.Integration;
+import io.sentry.core.SentryLevel;
+import io.sentry.core.SentryOptions;
+import java.lang.reflect.Method;
+
+final class NdkIntegration implements Integration {
+  private static boolean isNdkAvailable() {
+    return Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP;
+  }
+
+  @Override
+  public void register(IHub hub, SentryOptions options) {
```

Comment:
does it make sense to implement `Integration` if we don't use `hub` at all?

Reply:
maybe we may have another `interface` which is also an `Integration` only with `SentryOptions`, still respecting ISP principle.

Reply:
The idea of `Integration` is something that emit events, for that it requires `Hub` but in the case of NDK we're using reflection to get a type and initialize it and using the file system as a way to communicate with the SDK.

Reply:
I've added a comment to clarify